src/components/views/dashboard.tsx

```diff
@@ -76,30 +84,32 @@ export const Dashboard: FC<any> = ({navigation}) => {
     />
   );
 
+  const bluetoothOff = status.type?.indexOf(StatusType.bluetooth) !== -1;
+
   return (
     <Scrollable
       safeArea={false}
       toast={errorToast}
       backgroundColor={colors.background}
       refresh={{refreshing, onRefresh}}>
-      {exposure.tracingAvailable && (
+      {tracingAvailable && (
         <>
           <TracingAvailable ref={ref1} />
           <Spacing s={16} />
         </>
       )}
-      {exposure.contacts && exposure.contacts.length > 0 && (
+      {contacts && contacts.length > 0 && (
         <>
           <CloseContactWarning ref={ref2} />
           <Spacing s={16} />
         </>
       )}
-      {(!exposure.enabled ||
-        exposure.status.state !== StatusState.active ||
-        exposure.permissions.notifications.status !==
-          PermissionStatus.Allowed) && (
+      {(!enabled ||
+        status.state !== StatusState.active ||
+        permissions.notifications.status !== PermissionStatus.Allowed ||
+        bluetoothOff) && (
```

Comment:
No need to include `bluetoothOff` here. It's implied if ENS is not active. Just pass `status.type?.indexOf(StatusType.bluetooth) !== -1` to `AlertInformation` component.